OptimoveCore/Sources/Classes/NetworkClient/NetworkRequest.swift

```diff
@@ -87,17 +87,32 @@ public extension HTTPHeader {
     enum Fields: String {
         case contentType = "Content-Type"
         case userAgent = "User-Agent"
+        case tenantId = "X-Tenant-Id"
+        case accept
     }
 
-    enum Values: String {
-        case json = "application/json"
+    enum Values {
+        case json
+        case tenantId(id: String)
+        case textplain
+
+        var value: String {
+            switch self {
+            case .json:
+                "application/json"
+            case .tenantId(let id):
+                id
+            case .textplain:
+                "text/plain"
+            }
+        }
```

Comment:
Not sure if this was the correct way to implement the argument passing, let me know